src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -10,6 +10,7 @@ CA1047 | Design | Info | DoNotDeclareProtectedMembersInSealedTypes, [Documentati
 CA1069 | Design | Info | EnumShouldNotHaveDuplicatedValues, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1069)
 CA1070 | Design | Info | DoNotDeclareEventFieldsAsVirtual, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1070)
 CA1416 | Interoperability | Info | RuntimePlatformCheckAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1416)
+CA1417 | Interoperability | Warning | PlatformCompatabilityAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1417)
```

Comment:
In the real PR this will update CA1416 instead of adding new rule, i didn't want to update Manish's `RuntimePlatformCheckAnalyzer` in order to see and understand how flow analysis work and until to cover all the test cases it has. In future real PR those `RuntimePlatformCheckAnalyzer` will be updated/removed accordingly